netbox/core/views.py

```diff
@@ -560,6 +560,12 @@ def get(self, request):
             # Fall back to using the active config data if no record is found
             config = get_config()
 
+        # If Custom Validators is function (in configuration.py) get function name
+        if hasattr(config, 'CUSTOM_VALIDATORS') and config.CUSTOM_VALIDATORS:
+            for k, v in config.CUSTOM_VALIDATORS.items():
+                if isinstance(v, tuple):
```

Comment:
The type of every item in the list of configured validators needs to be checked.